### PR TITLE
Install the git version of wasm-bindgen

### PR DESCRIPTION
**Pull request by expenses**
This is needed for https://github.com/paritytech/substrate/pull/4887 @TriplEight

### CODE REVIEW

**Comment by TriplEight** [comment]
Test from https://github.com/paritytech/substrate/pull/4887 wasn't successful.

**Comment by TriplEight** [comment]
I've slightly changed everything

**Comment by General-Beck** [comment]
`geckodriver` release does not specify the `sha` amount, we leave it without verification

File: dockerfiles/substrate-ci-linux/Dockerfile [hunk 1]
```diff
@@ -18,23 +18,25 @@ dockerfiles/rust-builder/README.md" \
...
-		> /etc/apt/sources.list.d/mozilla.list; \
 	apt-get -y update; \
 	apt-get install -y --no-install-recommends \
-		firefox; \
+		firefox-esr; \
+# install geckodriver
+	curl -s https://api.github.com/repos/mozilla/geckodriver/releases/latest \
+		| grep 'browser_download_url.*linux64.tar.gz"' \
+		| cut -d : -f 2,3 | tr -d \" | xargs -n 1 curl -sSL \
+		| tar -xzC /usr/bin; \
```
Comment (gabreal) [inline dockerfiles/substrate-ci-linux/Dockerfile]: these kind of files should go to `/usr/loca/bin` in order not to override existing files. also `xargs -n 1` runs curl with one file at a time so should there be multiple files to download they will all be downloaded and unpacked to that folder. the easy way out would be to pipe it to `tail -n1` and use `sort -V` before. would it make sense to fix it to one version for reproducibilty reasons?

File: dockerfiles/rust-builder/Dockerfile [hunk 2]
```diff
@@ -34,14 +32,17 @@ ADD "https://static.rust-lang.org/rustup/dist/x86_64-unknown-linux-gnu/rustup-in
...
+		libssl-dev clang lld libclang-dev cmake make firefox-esr \
 		git pkg-config curl time rhash ca-certificates; \
+# install geckodriver
+	curl -s https://api.github.com/repos/mozilla/geckodriver/releases/latest \
+		| grep 'browser_download_url.*linux64.tar.gz"' \
+		| cut -d : -f 2,3 | tr -d \" | xargs -n 1 curl -sSL \
+		| tar -xzC /usr/bin; \
 # set a link to clang
-	ln -sf /usr/bin/clang /usr/bin/cc;\
+	ln -sf /usr/bin/clang /usr/bin/cc; \
```
Comment (gabreal) [inline dockerfiles/rust-builder/Dockerfile]: `update-alternatives --install /usr/bin/cc cc /usr/bin/clang 100` should be prefered

**Pull request merged**